_markbind/headers/header.md

```diff
@@ -1,6 +1,9 @@
 <header>
   <navbar type="dark">
     <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand"><img src="{{baseUrl}}/assets/catcher_logo.png" width="50"/></a>
+    <a href="{{baseUrl}}/index.html" class="navbar-brand"> Home </a>
+    <a href="{{baseUrl}}/ug/index.html" class="navbar-brand"> User Guide </a>
+    <a href="https://github.com/CATcher-org/CATcher" class="navbar-brand"> CATcher Repo </a>

```

Comment:
Could we have this look like the icon in here

![image](https://user-images.githubusercontent.com/39243082/114267692-c2f66480-9a2f-11eb-9e48-e4dc06dc6cc5.png)

Looks cooler 😅

Reply:
![Screenshot from 2021-04-10 22-49-35](https://user-images.githubusercontent.com/42912708/114274004-162bdf80-9a4f-11eb-88ed-e65587b72ce4.png)

Will this be Ok? :sweat_smile: